src/test/suite/fixtures/recorded/languages/typescript/chuckFunk.yml

```diff
@@ -0,0 +1,27 @@
+languageId: typescript
+command:
+  version: 1
+  spokenForm: chuck funk
```

Comment:
Not worth rewriting test, but we usually use "clear" / "change" unless removal is interesting (eg removes trailing comma or something)